evalml/model_understanding/prediction_explanations/explainers.py

```diff
@@ -63,12 +64,13 @@ def explain_predictions(pipeline, input_features, y, indices_to_explain, top_k_f
 
     report_creator = _report_creator_factory(data, report_type="explain_predictions",
                                              output_format=output_format, top_k_features=top_k_features,
-                                             include_shap_values=include_shap_values)
+                                             include_shap_values=include_shap_values,
+                                             include_expected_value=include_expected_value)
     return report_creator(data)
 
 
 def explain_predictions_best_worst(pipeline, input_features, y_true, num_to_explain=5, top_k_features=3,
-                                   include_shap_values=False, metric=None, output_format="text"):
+                                   include_shap_values=False, include_expected_value=False, metric=None, output_format="text"):
```

Comment:
I think there is a bug here because the expected value is included regardless of the value of this parameter

```python
exp = explain_predictions(pipeline=pipeline, input_features=X, y=None, indices_to_explain=[3],
                           top_k_features=6, include_shap_values=True, output_format="dict",
                   include_expected_value=False)
for explanation in exp["explanations"]:
    for class_explanation in explanation["explanations"]:
        assert class_explanation["expected_value"] != 0
```

I think in this case it's fine to not surface this parameter to the top-level and instead always include the expected value in the dict output. The expected value really only makes sense for the dict output format so I worry that including this parameter will cause users who use the dataframe or text output to be confused.